src/frontend/komponenter/Felleskomponenter/DødsfallTag.tsx

```diff
@@ -13,7 +13,7 @@ const DødsfallTag: React.FC<IDødsfallTagProps> = ({ dødsfallDato }) => {
         isoString: dødsfallDato,
         tilFormat: Datoformat.DATO,
     });
-    return <Tag variant="neutral-filled">{`Død ${formatertDato}`}</Tag>;
+    return <Tag variant="neutral-filled" size="small">{`Død ${formatertDato}`}</Tag>;
```

Comment:
Dette er for å gjøre tagen lik i størrelse som "Endret migreringsdato"-tagen som også ligger i personlinjen